src/main/java/calculator/controller/InputCalculator.java

```diff
@@ -0,0 +1,16 @@
+package calculator.controller;
+
+import calculator.model.Input;
+import java.util.List;
+
+public class InputCalculator {
+
+    public static int calculateResult(Input input) {
+        List<Integer> numbers = input.getNumbers();
```

Comment:
```suggestion
    public static int calculateResult(List<Integer> numbers) {
```
`Input`을 받아서 메서드에서 다시 나누는것보다 매개변수로 이미 나눠진 숫자들을 받아보는건 어떨까요??